dashboard/app/models/levels/external.rb

```diff
@@ -24,6 +24,9 @@
 #
 
 class External < DSLDefined
+  serialized_attrs %w(
+    display_as_unplugged
+  )
```

Comment:
I realize you may only part-way through responding to feedback, but I don't think this is the right approach. when you save an external level in levelbuilder, it goes down through the call stack like this:
* https://github.com/code-dot-org/code-dot-org/blob/41d8adc6a86afb0ff08b11f171fa881490736b72/dashboard/app/models/levels/dsl_defined.rb#L156-L158
* https://github.com/code-dot-org/code-dot-org/blob/41d8adc6a86afb0ff08b11f171fa881490736b72/dashboard/app/models/levels/dsl_defined.rb#L70-L74
* https://github.com/code-dot-org/code-dot-org/blob/f749e1c7cbb878c4af832879ed899a691549499e/dashboard/app/dsl/base_dsl.rb#L31-L34

from there, instance eval reads the individual commands in the dsl file (e.g. `display_as_unplugged true`) and calls the corresponding function on the DSL class. So, I believe what you need to do here is remove the serialized_attrs and instead implement a method like the one I posted earlier for teacher_markdown.

Reply:
thanks for the info, Dave!